Gems/Atom/Asset/Shader/Config/Platform/Android/shader_build_options.json

```diff
@@ -15,6 +15,7 @@
               // mobile because that decoration is not supported by most mobile drivers.
               , "-enable-16bit-types"
               , "-fvk-disable-depth-hint" // Disable image depth hint because it causes some crashes on mobile drivers.
+              , "-fspv-target-env=vulkan1.1"
```

Comment:
This should be added directly to the shader that needs this and not globally to all Android shaders without testing.  For example for the targeting .shader file (maybe DownsampleSinglePassLuminance.shader in this case) you can add this 

```
"AddBuildArguments": {
        "dxc": ["-fspv-target-env=vulkan1.1"]
    }
```

Reply:
I see, thanks!